MiniApp/Classes/Models/MiniAppJavascriptErrorInfo.swift

```diff
@@ -27,8 +25,8 @@ public enum MiniAppCustomPermissionError: String, Codable {
     }
 }
 
-func getMiniAppCustomPermissionError(customPermissionError: MiniAppCustomPermissionError) -> String {
-    return getErrorJsonResponse(error: MiniAppError(error: MiniAppErrorDetail(title: customPermissionError, description: customPermissionError.description)))
+func getMiniAppErrorMessage<T: MiniAppErrorProtocol>(_ error: T) -> String {
+    return getErrorJsonResponse(error: MiniAppError(error: MiniAppErrorDetail(title: error.name, description: error.message)))
```

Comment:
This isn't correct, is it? It should be returning the same format as a [Javascript Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) object: `{name: 'name', description: 'description'}`.

Reply:
Ah, I removed the `error` now, it will be `{name: 'name', description: 'description'}` now.

Reply:
This method is updated to return String as `ERROR_NAME: ERROR_DESCRIPTION`